### PR TITLE
Formdesign

### PR DESCRIPTION
**Pull request by vennela413**

### CODE REVIEW

File: form.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
 <p id="result"></p>
 <p id="word"></p>
 <p id="check"></p>
-<p id="date"></p>
+    <p id="date"></p>
```
Comment (svenkat683) [inline form.html:27]: What is change here? It is just space. You need to check these type mistakes also.

File: form.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+function myFunction() {
+    var z = document.getElementById("Name").value;
+    var y=document.getElementById("branch").value;
+    var pd = document.getElementById("Pass").value;
+    var x = document.getElementById("myDate").value;
+  var ele = document.getElementsByName("Gender");
```
Comment (svenkat683) [inline form.js:6]: I  don't like these types of naming conventions. I would rather prefer more appropriate names.

File: form.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    document.getElementById("myDate").value="";
+    for(i=0;i<=ele.length;i++){
+        if(ele[i].checked)
+        document.getElementById("check").innerHTML="Gender:"+ele[i].value;
+        ele[i].checked= false;
+   
+   }
+  
+  }
+
```
Comment (svenkat683) [inline form.js:23]: Where are you using this file?

File: formdesign.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.9.0/css/bulma.min.css">
```
Comment (svenkat683) [inline formdesign.html:1]: Why you are not using `<html>`, `<head>`, and `<body>` tags?

**Comment by svenkat683** [comment]
Make sure these changes working? Seems they won't work.

File: formdesign.html (hunk 4 above)
Comment (vennela413) [reply formdesign.html:1]: ok i will correct it.

File: form.js (hunk 3 above)
Comment (vennela413) [reply form.js:23]: that is previous pr file

File: form.js (hunk 2 above)
Comment (vennela413) [reply form.js:6]: okk

File: form.html (hunk 1 above)
Comment (vennela413) [reply form.html:27]: ok anna

File: form.js (hunk 3 above)
Comment (svenkat683) [reply form.js:23]: Remove it in this PR

**Pull request closed without merging**